projects/rero/ng-core/src/lib/record/record.module.ts

```diff
@@ -123,8 +123,8 @@ import { AggregationDateRangeComponent } from './search/aggregation/date-range/d
     BsDatepickerModule.forRoot(),
     FormlyModule.forRoot({
       extras: {
-        resetFieldOnHide: true,
-        lazyRender: true
+        resetFieldOnHide: false,
```

Comment:
Really? This will make problems when we hide/show a field.

Reply:
This is a mistake.